torchci/lib/bot/mergeBot.ts

```diff
@@ -46,19 +47,28 @@ function mergeBot(app: Probot): void {
       await dispatchEvent("try-revert");
       await reactOnComment(ctx, "+1");
     }
+    if (commentBody.match(rebaseCmdPat) && ctx.payload.comment.user.login == "clee2000") {
```

Comment:
nit: If we're going to add more and more commands here, might want to make some of these conditions an `else if` so that we don't actually have an intersect of commands that causes it to try revert and rebase on accident.